doc/api/os.md

```diff
@@ -2,25 +2,35 @@
 
     Stability: 2 - Stable
 
-Provides a few basic operating-system related utility functions.
+The `os` module provides a number of operating-system related utility methods.
+It can be accessed using:
 
-Use `require('os')` to access this module.
+```js
+const os = require('os');
+```
 
 ## os.EOL
 <!-- YAML
 added: v0.7.8
 -->
 
-A constant defining the appropriate End-of-line marker for the operating
-system.
+Returns a constant string defining the operating-system specific end-of-line
+marker:
+
+* `\n` on POSIX
+* `\r\n` on Windows
 
 ## os.arch()
 <!-- YAML
 added: v0.5.0
 -->
 
-Returns the operating system CPU architecture. Possible values are `'x64'`,
-`'arm'` and `'ia32'`. Returns the value of [`process.arch`][].
+The `os.arch()` method returns a string identifying the operating system CPU
+architecture.
+
+The current possible values are: `'x64'`, `'arm'` and `'ia32'`.
```

Comment:
FWIW [the current list from `configure`](https://github.com/nodejs/node/blob/84ad31fff369f0ec9a972c2a11cdefee316dab9d/configure#L32-L33) is:

* arm
* arm64
* ia32
* mips
* mipsel
* ppc
* ppc64
* s390
* s390x
* x32
* x64
* x86